tests/cephadm/test_nfs.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from ceph.ceph_admin.common import fetch_method
+from ceph.ceph_admin.nfs import NFS
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Ceph-admin module to manage NFS service
+
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): Ceph cluster object
+        kw: test data
+
+    check ceph.ceph_admin.nfs for test config
+    """
+    log.info("Running Ceph-admin Rados-GW( RGW ) test")
```

Comment:
This is a mistaken copy-and-paste from `tests/cephadm/test_rgw.py`, mind updating this?

Reply:
Addressed in https://github.com/red-hat-storage/cephci/pull/385